doc/release_notes_0.5.rst

```diff
@@ -0,0 +1,560 @@
+.. _release_notes_0.5:
+
+ACRN v0.5 (Jan 2019)
+####################
+
+We are pleased to announce the release of Project ACRN version 0.5.
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline
+embedded development through an open source platform. Check out the
+:ref:`introduction` for more information.
+
+
+All project ACRN source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, and documentation.
+You can either download this source code as a zip or tar.gz file (see
+the `ACRN v0.5 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v0.5>`_ or
+use Git clone and checkout commands:
+
+.. code-block:: bash
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v0.5
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v0.5 documents can be found at
+https://projectacrn.github.io/0.5/.  Documentation for the latest
+(master) branch is found at https://projectacrn.github.io/latest/.
+
+
+Version 0.5 new features
+************************
+
+**OVMF support initial patches merged in ACRN**: 
+In order to boot Guest Windows on ACRN, the virtual machines firmware is 
+suggested to use OVMF, which is Opensource Virtual Machine Firmware. 
+Initial patches to support OVMF have been merged in ACRN hypervisor. 
+More patches for ACRN and patches upstreaming to OVMF work will be continuing.
+
+**UP2 board serial port support**: 
+Enabled UP2 board’s serial port debugging during SOS and UOS boot.
+One E2E binary to support all UEFI platform: ACRN can support 
```

Comment:
This looks like it may be new item.  If so change to:
```
**One E2E binary to support all UEFI platform**: 
ACRN can support both ApolloLake (APL) and KabbyLake (KBL) NUCs.
Instead of having separate builds, this release offers community
developers a single end-to-end reference build that supports both
UEFI hardware platforms, configured with a new boot parameter.
See :ref:`getting_started` for more information.
```